### DISCUSSION TITLE
Executeable files for Windows user?

### DISCUSSION THREAD

**Discussion by NightSkyYozora** [sherlock-project/sherlock#937] [Ideas]
Can you build standalone executeable files for Windows user? I don't own Linux machine and only been running it on repl.it

It would be nice if there is a standalone version, the question is, is it possible to work on? or simply can't be build because of the nature of sherlock?

Thanks!

**Comment by hoadlck** [comment]
Sherlock works fine on Windows.  You do need to have Git and Python installed, but it will work.

It certainly is technically possible to make an executable.  But, the priority is to get Sherlock packaged like a normal Python package.

**Reply by 404oops** [reply]
> Sherlock works fine on Windows. You do need to have Git and Python installed, but it will work.
> 
> It certainly is technically possible to make an executable. But, the priority is to get Sherlock packaged like a normal Python package.

You don't need git, you can download the repo as zip

**Comment by sagarspatil** [comment]


**Reply by vijfkebabs** [reply]
nice malware link buddy

**Reply by ppfeister** [reply]
@vijfkebabs I believe it was legit back in 2021, but after five years, someone probably snagged the domain

I've removed the comment based on your mention. A small handful of providers have it flagged as malicious now.

**Reply by vijfkebabs** [reply]
yea probably. thanks for deleting it so people cant get their data stolen
